Fixes Issue #5843 (#5983)
umbraco/UmbracoDocs: 13/umbraco-cms/tutorials/add-google-authentication.md

When you log in to the Umbraco Backoffice, you need to enter your username and password. Integrating your website with Google authentication adds a button that you can click to log in with your Google account.

> ![Google login screen](images/googleLoginScreen_v13.png.png)

Reviewer: ```suggestion
![Google login screen](images/googleLoginScreen_v13.png)
```

1. Click the project dropdown and select **New Project**.

    ![Project dropdown list](images/Project_dropdown_list_v13.png)

2. Enter a **Project name**, **Organization**, and **Location**.
3. Click **Create**.

### Enable the Google+ API

1. Open the newly created project from the project dropdown.
2. Click **Enable APIs and Services**.

    ![Enable APIs](images/Enable_Apis_v13.png)

3. Type **Google+ API** in the **Search** field.

> 4. Click **Enable**.

Reviewer: just checked and first you need to click on it and only after you get the choice to enable it
```suggestion
4. Select it and then **Enable** it.
```

    ![Select User Type](images/User_Type_v13.png)

4. Fill in the required information:
   * App name
   * User support email
   * Developer contact information
5. Click **Save and Continue**.
6. Select the scopes your project needs.
7. Click **Save and Continue**.
8. Verify the details you have provided.
9. Click **Back to Dashboard** to complete creating the Consent screen.

### Create credentials

1. Click **Credentials** from the left-side navigation menu.
2. Click **Create Credentials**.
3. Select **OAuth Client ID** from the dropdown.

    ![Select OAuth Client ID](images/OAuth_Client_Id_v13.png)

4. Select **Web Application** from the **Application type** dropdown.
5. Enter the following details:
   * Application **Name**
   * **Authorized JavaScript origins**
   * **Authorized redirect URIs**

    ![Credentials](images/credentials_v13.png)

6. Click **Create**.

A popup appears displaying the **Client Id** and **Client Secret**. You will need these values later while configuring your solution.

{% hint style="info" %}
The **Client Id** and **Client Secret** can always be accessed from the **Credentials** tab in the **APIs & Services** menu.
{% endhint %}

> ## 2. Integrating Google Auth in Visual Studio

Reviewer: ```suggestion
## 2. Integrating Google Auth in your project
```

### Installing a Nuget Package

You can install and manage packages in a project.

> 1. Navigate to your project/solution folder.

Reviewer: ```suggestion
1. Navigate to your project/solution folder.

{% hint style="info" %}
If you have cloned down an Umbraco Project, you will need to navigate to the `src ` folder where you can see a `.csproj` file.
{% endhint %}
```

> 2. Open a command prompt of your choice.

Reviewer: they are called "command-lines" or command-line interface (CLI) https://attuneops.io/difference-between-cmd-vs-powershell-vs-bash/#:~:text=CMD%20is%20the%20command%20line,%2FLinux%2Dbased%20operating%20systems.
```suggestion
2. Open a command-line of your choice such as "Command Prompt" at the mentioned location .
```

3. Run the following command to install the `Microsoft.AspNetCore.Authentication.Google` package.

    ```cli

>     dotnet add package Microsoft.AspNetCore.Authentication.Google --version 8.0.3

Reviewer: just tested and it works without mentioning the version. If it is like I am suggesting here, it will always install the latest version. you can specify the version if you want to install older versions. Perhaps this can be changed in the info note below?
```suggestion
    dotnet add package Microsoft.AspNetCore.Authentication.Google
```

    ```

4. Once the package is installed, open the **.csproj** file to ensure if the package reference is added:

    ```js
    <ItemGroup>
        <PackageReference Include="Microsoft.AspNetCore.Authentication.Google" Version="8.0.3" />
    </ItemGroup>
    ```

{% hint style="info" %}

> Always check the [latest version of the package](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Google) before installing it.

Reviewer: ```suggestion
You can check the [latest version of the package](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Google) before installing it.
```
